src/libraries/System.Private.CoreLib/src/System/Text/EncodingTable.cs

```diff
@@ -229,19 +229,32 @@ private static CodePageDataItem InternalGetCodePageDataItem(int codePage, int in
             // All supported code pages have identical header names, and body names.
             string headerName = webName;
             string bodyName = webName;
-            string displayName = GetDisplayName(codePage, index);
+            string displayName = GetDisplayName(codePage);
             uint flags = Flags[index];
 
             return new CodePageDataItem(uiFamilyCodePage, webName, headerName, bodyName, displayName, flags);
         }
 
-        private static string GetDisplayName(int codePage, int englishNameIndex)
+        private static string GetDisplayName(int codePage)
         {
-            string? displayName = SR.GetResourceString("Globalization_cp_" + codePage.ToString());
-            if (string.IsNullOrEmpty(displayName))
```

Comment:
I think this was dead code. The only situation when GetResourceString could return a null is if we failed to find the manifest resource blob.

I traced this back to https://github.com/dotnet/corert/commit/db4c4e1a8a914155d74ac4eb7feedb61fda4a00d "this implementation does not try to find resource strings for EncodingName, because ProjectN does not support this yet". 6 years passed. We added support at some point.